### PR TITLE
Revert pandas

### PR DESCRIPTION
**Pull request by Oddant1**

### CODE REVIEW

**Comment by ebolyen** [comment]
I'm going to let busywork prove these tests, as we haven't had an up to date env file for travis in forever.

File: q2_sample_classifier/utilities.py [hunk 1]
```diff
@@ -694,7 +694,7 @@ def _mean_feature_importance(importances):
     containing importance scores of the same features from multiple models
     (e.g., CV importance scores).
     '''
-    imp = pd.concat(importances, axis=1, sort=True)
+    imp = pd.concat(importances, axis=1)
```
Comment (thermokarst) [inline q2_sample_classifier/utilities.py]: Do we need to revert this change?

Comment (ebolyen) [reply q2_sample_classifier/utilities.py]: perhaps not, but I told @Oddant1 to just create a revert commit

Comment (ebolyen) [reply q2_sample_classifier/utilities.py]: my thinking was we could double-revert later
